tests/pkg_cryptoauthlib_internal-tests/main.c

```diff
@@ -23,7 +23,12 @@
 int main(void)
 {
     /* Set device to ATECC508A */
-    atca_run_cmd("508");
+    if (ATCA_DEVTYPE == ATECC608A) {
+        atca_run_cmd("608");
+    }
+    else {
+        atca_run_cmd("508");
+    }
```

Comment:
A better solution would have been something like: `if(608), else if (508), else: not yet supported`  as the library theoretically also supports predecessors of that chip.